src/just_tls.erl

```diff
@@ -231,3 +235,14 @@ receive_verify_results(Acc) ->
 error_to_list(_Error) ->
     %TODO: implement later if needed
     "verify_fun_callback failed".
+
+session_tickets_opt(server, SslOpts, _Opts = #{session_tickets := ST})
+        when ST =:= stateless ->
+    [{session_tickets, ST} | SslOpts];
+session_tickets_opt(_ClientOrServer, SslOpts, _Opts) ->
+    SslOpts.
+
+early_data_opt(server, SslOpts, #{early_data := true}) ->
+    [{early_data, enabled} | SslOpts];
+early_data_opt(_ClientOrServer, SslOpts, _Opts) ->
+    SslOpts.
```

Comment:
I'd put these new `_opt` helpers just after all the other `_opt` helpers, to keep them all together.